.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,44 @@
+{
+	"name": "Recommenders",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"context": "..",
+		"args": { 
+			// Python version: 3, 3.6, 3.7
+			"PYTHON_VERSION": "3.7",
```

Comment:
@gramhagen have you tried this working with 3.7? We had some problems with it and had to install numpy-base, see https://github.com/microsoft/recommenders/pull/1520. With python 3.6, we didn't get any installation problem

Reply:
yes, interestingly there is no problem installing the package with python 3.7 here. curious, maybe it has to do with the os version being used?

Reply:
this is interesting, FYI @anargyri

Reply:
Yes, interesting indeed. Do you know what is the environment they use under the hood? (conda, venv, virtualenv?)